webrtc-stats.html

```diff
@@ -768,6 +768,7 @@
              double             burstDiscardRate;
              double             gapLossRate;
              double             gapDiscardRate;
+             DOMHighResTimeStamp  lastPacketReceivedTimestamp;
```

Comment:
@vr000m Does this merit a disclaimer similar to the one in [remoteTimestamp](https://w3c.github.io/webrtc-stats/#dom-rtcoutboundrtpstreamstats-remotetimestamp) for the case where this represents remote time?